src/WhatsNewDialog.cpp

```diff
@@ -39,13 +39,21 @@ namespace
 #  define SHOW_MUSEHUB
 #endif
 
-const char* WhatsNewURL = "https://audacityteam.org/3.6.0-video";
+const char* WhatsNewURL = "https://audacityteam.org/3.7.0-video";
+const char* ChangeLogURL = "https://support.audacityteam.org/additional-resources/changelog";
 const char* MuseHubURL = "https://www.musehub.com";
+const char* PromoURL = "https://audacityteam.org/audacitypromo";
 
 #if defined(SHOW_MUSEHUB)
-   constexpr auto WindowWidth = 720;
+   constexpr auto WindowWidth = 860;
 #else
-   constexpr auto WindowWidth = 400;
+   constexpr auto WindowWidth = 440;
+#endif
+
+#if defined(__WXOSX__)
+   constexpr auto WindowHeight = 400;
+#else
+   constexpr auto WindowHeight = 460;
 #endif
```

Comment:
```suggestion
constexpr auto WindowHeight = 460;
```

Reply:
it is also a bit too short on windows

![image](https://github.com/user-attachments/assets/eb8510c8-a520-43c8-b96e-7efd5bbca35b)

Reply:
Nevermind, it fits okay, when the version string does not have `alpha` suffix. I believe that adjusting the dialog height automatically depending on the textbox size is not a trivial task with wxWidgets.